charts/admission-controller/README.md

```diff
@@ -70,6 +70,7 @@ Controller chart and their default values:
 | `scanner.podAnnotations`              | Scanner pod annotations                                      | `{"prometheus.io/scrape": "true", "prometheus.io/path": "/metrics", "prometheus.io/port": "5000", "prometheus.io/scheme": "https"}` |
 | `scanner.psp.create`                  | Whether to create a psp policy and role / role-binding       | `false`                                                                                                                             |
 | `scanner.resources`                   | Resource requests and limits for scanner                     | `{}`                                                                                                                                |
+| `scanner.verifyRegistrySSL`           | Verify SSL on image pull from registries                     | `true`                                                                                                                              |
```

Comment:
I would prefer using TLS over SSL all over the place

Reply:
Wanted to keep it simmilar to `verifySSL` but will change it to `verifyRegistryTLS` :)